packages/contract_templates/contract.handlebars

```diff
@@ -1,7 +1,3 @@
-/**
```

Comment:
Was there a reason to remove this comment block?

Reply:
Yes, I explained it in the PR this was cloned from, see https://github.com/0xProject/0x-monorepo/pull/699#discussion_r195505551

Reply:
In short, I think the comment is misleading, because you *should* edit this file, per https://github.com/0xProject/0x-monorepo/tree/v2-prototype/packages/abi-gen#how-to-write-custom-templates

Reply:
Ah ok I missed that comment thread